src/PlusDataCollection/WinProbe/vtkPlusWinProbeVideoSource.cxx

```diff
@@ -1073,6 +1073,11 @@ void vtkPlusWinProbeVideoSource::SetBRFEnabled(bool value)
 {
   if(Connected)
   {
+    if(GetMModeEnabled())
```

Comment:
Calling this class' setters in both of these setters leads to infinite recursion. Did you try this? Why not just cherry-pick my commit(s)?

Reply:
Yes, you're right @dzenanz, I've cherry-picked your commits and updated this PR. Thanks!